helm/superset/README.md

```diff
@@ -142,6 +145,7 @@ helm install my-superset superset/superset
 | supersetCeleryFlower.replicaCount | int | `1` |  |
 | supersetCeleryFlower.resources | object | `{}` | Resource settings for the CeleryBeat pods - these settings overwrite might existing values from the global resources object defined above. |
 | supersetCeleryFlower.service.annotations | object | `{}` |  |
+| supersetCeleryFlower.service.nodePort.http | string | `""` |  |
```

Comment:
The type of nodePort should be `int` with a default of `nil`

Reply:
Done, thanks!